proof/fuzz_test.go

```diff
@@ -0,0 +1,31 @@
+package proof
+
+import (
+	"bytes"
+	"testing"
+)
+
+func FuzzFile(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		fileData := make([]byte, 0)
+		fileData = append(fileData, FilePrefixMagicBytes[:]...)
```

Comment:
nit: if we're not pre-allocating `fileData`, might as well do: `fileData := append([]byte{}, FilePrefixMagicBytes[:]...)` directly (here and below).